project_generator/tools/uvision.py

```diff
@@ -294,6 +294,7 @@ def _set_target(self, expanded_dic, uvproj_dic, tool_name):
             uvproj_dic['Project']['Targets']['Target']['TargetOption']['TargetCommonOption']['Cpu'] = mcu_def_dic['TargetOption']['Cpu'][0].encode('utf-8')
             uvproj_dic['Project']['Targets']['Target']['TargetOption']['TargetCommonOption']['FlashDriverDll'] = str(mcu_def_dic['TargetOption']['FlashDriverDll'][0]).encode('utf-8')
             uvproj_dic['Project']['Targets']['Target']['TargetOption']['TargetCommonOption']['SFDFile'] = mcu_def_dic['TargetOption']['SFDFile'][0]
+            uvproj_dic['Project']['Targets']['Target']['TargetOption']['TargetCommonOption']['RegisterFile'] = mcu_def_dic['TargetOption']['RegisterFile'][0]
```

Comment:
As I understand, registerfile was added just for uvision5? That means not all targets define registerfile. Which is ok, but the message below is not correct. TODO should be removed (as it's obsolete). Thus it should just continue.